.storybook/main.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
     "@storybook/addon-links",
     "@storybook/addon-a11y",
   ],
+  babel: () => {},
```

Comment:
storybook combines our config and their config which leads to the problem of resolving packages